### ISSUE TITLE
Broken download links for old images.

### ISSUE THREAD

**Issue by ghost** [midi1996/hackintosh-internet-install-gitbook#10]
The documentation listed the old links to the images.

https://internet-install.gitbook.io/macos-internet-install/preparing-your-installer/untitled-1

**Comment by midi1996** [comment]
Yes, you can try adding `-m 10.9` for 10.9 image (Developer Preview) `-m 10.10` for yosemite image and so on.

**Comment by ghost** [comment]
Maybe then it's worth making a change in the documentation?

**Comment by midi1996** [comment]
Maybe 🤔 but now im trying to merge this into the opencore guide, will probably drop this one.
